CHANGELOG.md

```diff
@@ -12,11 +12,10 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 ### Added - Unreleased
 
 - SC-6738 Extend search input field in new admin tables to search for full name
+- SC-6830 - Added hook to parse request to arrays when > 20 users are requested in adminUsers service
 
 ### Removed
 
-### Removed - Unreleased
-
```

Comment:
if possible please keep the Unreleased, makes it later easier to add the version number ;)